<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmAboutVP</class>
 <widget class="QMainWindow" name="fmAboutVP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>О программе</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lwAbout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actTIns">
   <property name="text">
    <string>Вставить строку</string>
   </property>
  </action>
  <action name="actTDel">
   <property name="text">
    <string>Удалить строку</string>
   </property>
  </action>
  <action name="actPrIns">
   <property name="text">
    <string>Новый профиль</string>
   </property>
  </action>
  <action name="actPrDel">
   <property name="text">
    <string>Удалить профиль</string>
   </property>
  </action>
  <action name="actPrRen">
   <property name="text">
    <string>Переименовать профиль</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
